src/bin/zmjava

```diff
@@ -34,7 +34,7 @@ CYGWIN*) PATHSEP=";";;
 *) PATHSEP=":";;
 esac
 
-ZIMBRA_EXTENSIONS="backup clamscanner network zimbra-license zimbrahsm zimbrasync twofactorauth com_zimbra_ssdb_ephemeral_store zimbra-archive"
+ZIMBRA_EXTENSIONS="backup clamscanner network zimbra-license zimbrahsm zimbrasync twofactorauth com_zimbra_ssdb_ephemeral_store zimbra-archive zm-modules-porter"
```

Comment:
does this mean extension will be always loaded and available? although customer is going to use it once in lifetime?

Reply:
it will be load if it is installed

Reply:
Hope it's not going to flood logs with errors of module not found